src/modal/modal.js

```diff
@@ -547,6 +547,16 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
         return false;
       };
 
+      $modalStack.isFocusInModal = function(evt, modalWindow) {
+        if (evt && modalWindow) {
+          var modalDomEl = modalWindow.value.modalDomEl;
+          if (modalDomEl && modalDomEl.length) {
+            return (evt.target || evt.srcElement) === modalDomEl[0];
```

Comment:
But by using [contains](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains), method will return `true` for each tabbable element within modalWindow